velox/expression/Expr.h

```diff
@@ -63,25 +63,64 @@ struct ExprStats {
   }
 };
 
+class ExprSaver {
+ public:
+  /// Persist the passed in 'RowVector' on disk. The data will be persisted at
+  /// $basePath/velox_vector_xxx that will be returned by storedPath.
+  /// Returns true if persistence is successful, false otherwise. In the case of
+  /// unsuccessful persistence, the reason will be returned by failMsg.
+  static bool saveData(
```

Comment:
Do we need to add more functionality to expression evaluation engine. Can this logic be added to the Fuzzer directly?

Reply:
ExprExpcetionContext needs this logic as well, which is in Expr.h